docs/memcached.md

```diff
@@ -0,0 +1,14 @@
+# Memchached entries
```

Comment:
Some basic docs with memcached entries we use (at least those that I touched in this PR).